ledger-tool/src/main.rs

```diff
@@ -1238,19 +1241,38 @@ fn load_bank_forks(
         None,
     );
 
+    let (transaction_status_sender, transaction_status_service) = if transaction_notifier.is_some() {
+        let (transaction_status_sender, transaction_status_receiver) = unbounded();
+        let transaction_status_service = TransactionStatusService::new(
+            transaction_status_receiver,
+            Arc::default(),
+            false,
+            transaction_notifier,
+            blockstore.clone(),
+            false,
+            &exit.clone(),
```

Comment:
remove unnecessary clone -- it is cloned inside the callee. CI also complains about it.